tests/unit/ui/components/TestExpandableCard.js

```diff
@@ -0,0 +1,59 @@
+import { mount } from 'enzyme';
+import React from 'react';
+
+import ExpandableCard from 'ui/components/ExpandableCard';
+import { createFakeEvent, getFakeI18nInst } from 'tests/unit/helpers';
+
+
+function render(props) {
+  // We have to mount because of the usage of `setState` in this component.
```

Comment:
It looks like you only need `mount()` for the interaction with `Card` and only for one test: *it toggles when clicked*. I think it would be good to use `shallowUntilTarget()` for the other tests to keep it lean.

Reply:
Ah, fair enough. I guess there's probably a decent speed difference when all the tests are considered in using `shallow` whenever possible.

Reply:
In addition to speed, `mount()` also makes tests brittle since a problem with a component farther down in the render tree could cause the test to fail.

Reply:
Ah, true.